app/uk/gov/hmrc/thirdpartyapplication/config/ConfigurationProviders.scala

```diff
@@ -138,12 +107,11 @@ class ApiStorageConfigProvider @Inject()(val runModeConfiguration: Configuration
   override protected def mode = environment.mode
 
   override def get() = {
-    val skipWso2 =
-      runModeConfiguration.getBoolean(s"$env.skipWso2").
-      getOrElse(runModeConfiguration.getBoolean("skipWso2").
+    val disableAwsCalls =
+      runModeConfiguration.getBoolean(s"$env.disableAwsCalls").
+      getOrElse(runModeConfiguration.getBoolean("disableAwsCalls").
         getOrElse(false))
```

Comment:
Is this to default to 'false'?

Reply:
Correct - we want it switched on by default